### DISCUSSION TITLE
Higher zoom and no image smoothing

### DISCUSSION THREAD

**Discussion by fellbyyourgun** [wkentaro/labelme#1128] [Ideas / Feature requests]
I'm working on a project with very small features inside of very big images that need labelling.  Features may be 3 or 4 pixels wide at some points in an image that is 4000x4000 pixels.  It would be nice if I could zoom to say, 2000 percent instead of 1000 percent.  

Image smoothing also makes it difficult to correctly place the vertices for polygons correctly.  It would be nice to be able to zoom in without smoothing so that I can place the vertices while viewing the actual pixels.

**Comment by jkfindeisen** [comment]
One temporary workaround would be to cut the input images (4000 x 4000) into smaller ones before labelling and processing (many neuronal nets will have memory problems or inbuilt downsampling anyway). With, say 2000x2000, or smaller images this problem would be less severe.

**Comment by mr-jstar** [comment]
I've solved both issues. However, my code is pretty dirty: I work with thermographic images and have added some functionality related to these. If you are interested, can give you the code with my corrections.

**Reply by wkentaro** [reply]
Can you post a link to your solution? Possibly I (or someone) can fork that code to include in this repo!

**Comment by mr-jstar** [comment]
Sure (although I’m ashamed withe quality of my code ;)
https://cloud.ee.pw.edu.pl/nextcloud/index.php/s/AMkytNX4XacTaoK
Regards,
Jacek Starzyński

**Reply by wkentaro** [reply]
Great!
